pkg/admission/test/unit/admission_unit_test.go

```diff
@@ -405,124 +353,98 @@ var _ = Describe("NamespaceStore admission unit tests", func() {
 							},
 						},
 					}
-					result, message = nsv.ValidateNSEmptySecretName()
-					Expect(result).To(BeTrue())
-					Expect(message).To(Equal("allowed"))
-				})
-			})
-			Context("Invalid store type", func() {
```

Comment:
why all the invalid store type tests were deleted?

Reply:
re-added them :)